src/error/__tests__/GraphQLError-test.ts

```diff
@@ -353,4 +353,29 @@ describe('toJSON', () => {
       extensions: { foo: 'bar' },
     });
   });
+
+  it('can be created with the alternative object argument', () => {
+    const error = new GraphQLError('msg', {
+      nodes: [operationNode],
+      source,
+      positions: [6],
+      path: ['path', 2, 'a'],
+      originalError: new Error('I like turtles'),
+      extensions: { hee: 'I like turtles' },
+    });
+
+    expect(error.toJSON()).to.deep.equal({
+      extensions: {
```

Comment:
please order fields based on the order of `GraphQLError` properties in definitions.
Please compact short properties into a single line as with other tests in that file.

Reply:
done!